From 5631535841f16f3c54cfeedd518f4ffa4f29fb78 Mon Sep 17 00:00:00 2001
From: wulan17 <galihgustip@gmail.com>
Date: Thu, 22 Oct 2020 17:16:19 +0000
Subject: [PATCH] Revert "exfat fsck/mkfs selinux label"

This reverts commit df25576594f684ed35610b7cc1db2b72bc1fc4d6.

Signed-off-by: wulan17 <galihgustip@gmail.com>

diff --git a/sepolicy/file_contexts b/sepolicy/file_contexts
index 4ae5095..5e39d5b 100644
--- a/sepolicy/file_contexts
+++ b/sepolicy/file_contexts
@@ -6,9 +6,6 @@
 /system/bin/phh-on-data.sh u:object_r:phhsu_exec:s0
 /system/bin/asus-motor u:object_r:phhsu_exec:s0
 
-#/system/bin/fsck\.exfat                 u:object_r:fsck_exec:s0
-/system/bin/fsck\.ntfs                  u:object_r:fsck_exec:s0
-
 /bt_firmware(/.*)?      u:object_r:bt_firmware_file:s0
 
 /sec_storage(/.*)?      u:object_r:teecd_data_file:s0
-- 
2.25.1

