From a8632314259d3354c5bcd811007511f2a1429460 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Wed, 11 Mar 2020 14:02:35 +0100
Subject: [PATCH] Every voldmanaged storage is adoptable


diff --git a/main.cpp b/main.cpp
index ebe5510..113d9b9 100644
--- a/main.cpp
+++ b/main.cpp
@@ -249,7 +249,7 @@ static int process_config(VolumeManager* vm, bool* has_adoptable, bool* has_quot
             std::string nickname(entry.label);
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) {
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 *has_adoptable = true;
             }
-- 
2.25.1

